lnbits/core/views/api.py

```diff
@@ -657,3 +659,23 @@ async def _generator(stream: BytesIO):
             "Expires": "0",
         },
     )
+
+
+@core_app.get("/api/v1/audit/")
+async def api_auditor(wallet: WalletTypeInfo = Depends(get_key_type)):
+    if wallet.wallet.user not in LNBITS_ADMIN_USERS:
+        raise HTTPException(
+            status_code=HTTPStatus.FORBIDDEN, detail="Not an admin user"
+        )
+
+    total_balance = await get_total_balance()
```

Comment:
If there are no balances, then `total_balance` is `None`. It should be `zero`.
Can be reproduced with `LNBITS_BACKEND_WALLET_CLASS=FakeWallet`

```
delta = node_balance - total_balance
TypeError: unsupported operand type(s) for -: 'int' and 'NoneType'
```